tests/integration/test_legacy_search_objects.py

```diff
@@ -33,28 +33,6 @@ def test_search_example3(service):
     assert len(res['objects']) > 0
 
 
-def test_search_example4(service):
```

Comment:
Any reason these were dropped?

Reply:
They test GenomeFeatures, which are not supported now
In the PR comments I indicated that I had stripped out GenomeFeature custom support from the legacy api.
It would better to add that support later when we have a solid use case for it, and can do so generically without having custom switches in the code to support just a single index which we don't currently use.
Of course, better minds may disagree, so we can discuss!

Reply:
There is an expense to maintaining quality tests for features, so if a feature isn't used, it seems most practical to remove the tests rather than to fix them.
This was driven by the fact that I needed to add coverage for GenomeFeatures, and these tests were not passing, so it seemed the most prudent action.

Reply:
Sounds fine.